tools/cli/NearObjectCli.cxx

```diff
@@ -227,22 +232,33 @@ NearObjectCli::AddSubcommandUwbRangeStart(CLI::App* parent)
+    // TODO: Accept multiple controlees
+    if (uwbConfig.macAddressMode == uwb::UwbMacAddressType::Extended) {
+        rangeStartApp->add_option("--DeviceMacAddress", m_cliData->deviceMacAddress, "8-byte extended MAC address of own device: e.g. 12:34:56:78:87:65:43:21")->capture_default_str()->required();
+        rangeStartApp->add_option("--DestinationMacAddress", m_cliData->destinationMacAddress, "8-byte extended MAC address of other device(s). If device is Controller, list NumberOfControlees mac addresses. If device is Controlee, list Controller mac address")->capture_default_str()->required();
+    } else { // uwb::UwbMacAddressType::Short OR empty (default)
+        rangeStartApp->add_option("--DeviceMacAddress", m_cliData->deviceMacAddress, "2-byte short MAC address of own device: e.g. 12:34")->capture_default_str()->required();
+        rangeStartApp->add_option("--DestinationMacAddress", m_cliData->destinationMacAddress, "2-byte short MAC address of controlee. If device is Controller, list NumberOfControlees mac addresses. If device is Controlee, list Controller mac address")->capture_default_str()->required();
+    }
+    detail::AddEnumOption(rangeStartApp, uwbConfig.deviceType, true);
+
     // enumerations
-    detail::AddEnumOption(rangeStartApp, uwbConfig.deviceRole);
-    detail::AddEnumOption(rangeStartApp, uwbConfig.rangingDirection);
-    detail::AddEnumOption(rangeStartApp, uwbConfig.rangingMeasurementReportMode);
-    detail::AddEnumOption(rangeStartApp, uwbConfig.stsConfiguration);
-    detail::AddEnumOption(rangeStartApp, uwbConfig.multiNodeMode);
-    detail::AddEnumOption(rangeStartApp, uwbConfig.rangingTimeStruct);
-    detail::AddEnumOption(rangeStartApp, uwbConfig.schedulingMode);
-    detail::AddEnumOption(rangeStartApp, uwbConfig.channel);
-    detail::AddEnumOption(rangeStartApp, uwbConfig.rframeConfig);
-    detail::AddEnumOption(rangeStartApp, uwbConfig.convolutionalCodeConstraintLength);
-    detail::AddEnumOption(rangeStartApp, uwbConfig.prfMode);
-    detail::AddEnumOption(rangeStartApp, uwbConfig.macAddressFcsType);
+    detail::AddEnumOption(rangeStartApp, uwbConfig.rangingDirection, false);
+    detail::AddEnumOption(rangeStartApp, uwbConfig.rangingMeasurementReportMode, false);
+    detail::AddEnumOption(rangeStartApp, uwbConfig.stsConfiguration, false);
+    detail::AddEnumOption(rangeStartApp, uwbConfig.rangingTimeStruct, false);
+    detail::AddEnumOption(rangeStartApp, uwbConfig.schedulingMode, false);
+    detail::AddEnumOption(rangeStartApp, uwbConfig.channel, false);
+    detail::AddEnumOption(rangeStartApp, uwbConfig.rframeConfig, false);
+    detail::AddEnumOption(rangeStartApp, uwbConfig.convolutionalCodeConstraintLength, false);
+    detail::AddEnumOption(rangeStartApp, uwbConfig.prfMode, false);
+    detail::AddEnumOption(rangeStartApp, uwbConfig.macAddressFcsType, false);
```

Comment:
Instead of specifying `false` for the majority of these, why not provide a default `true` value for the `isMandatory` argument. This way, the arg is only specified for the exceptional cases and makes the code less verbose.